crossing-training.lic

```diff
@@ -314,6 +315,28 @@ class CrossingTraining
       fput('cast')
     end
 
+    if data['moon']
+      moon = UserVars.moons['visible'].first
+      unless moon
+        echo "No moon available to cast #{data['abbrev']}"
+        unless Script.running? 'moonwatch'
+          echo "moonwatch is not running. Starting it now"
+          custom_require 'moonwatch'
+          echo "Run `;e autostart('moonwatch')` to avoid this in the future"
+        end
+        weather = bput('weather', "inside", "You glance up at the sky.")
+        if weather =~ /inside/
+          walk_to @settings.outdoor_room
+        end
+        fput('perceive moons')
```

Comment:
Probably not, no. I didn't have a full appreciation for moonwatch when I wrote this.